RealmSwift-swift2.2/Tests/RealmTests.swift

```diff
@@ -255,6 +255,29 @@ class RealmTests: TestCase {
         }
         XCTAssertEqual(try! Realm().objects(SwiftStringObject.self).count, 0)
     }
+    
+    func testThrowsWrite() {
+        enum TestError: ErrorType { case Exception }
```

Comment:
Apparently `assertFails` requires that I throw a `Realm.Error` rather than any `ErrorType`. I'm not sure the best way to test this. If I were to throw an existing `Error` case, I couldn't be sure that some unrelated failure didn't occur.